docs/sources/reference/run.md

```diff
@@ -394,9 +394,47 @@ command:
 
     # docker run --security-opt label:type:svirt_apache_t -i -t centos bash
 
-Note:
-
-You would have to write policy defining a `svirt_apache_t` type.
+> **Note**: You would have to write policy defining a `svirt_apache_t` type.
+
+## Volume mounts and SELinux
```

Comment:
If your host is using SELinux (Security-Enhanced Linux), you can mount volumes
on a container but the container cannot write to them. Attempts to
write to a mounted volume result in a  `Permission denied` message and an `avc:` message
in the host's `syslog`. 

   [root@RHEL71 sven]# docker run -v /var/db:/data1 -i -t fedora ls -laZ /data1
   ls: cannot access /data1/Makefile: Permission denied
   drwxr-xr-x. root root system_u:object_r:var_t:s0       .
   drwxr-xr-x. root root system_u:object_r:svirt_sandbox_file_t:s0:c602,c707 ..
   ?---------  ?    ?                                     Makefile
   drwx------. root root system_u:object_r:sudo_db_t:s0   sudo
   [root@RHEL71 sven]# ls -laZ /var/db
   drwxr-xr-x. root root system_u:object_r:var_t:s0       .
   drwxr-xr-x. root root system_u:object_r:var_t:s0       ..
   -rw-r--r--. root root system_u:object_r:var_t:s0       Makefile
   drwx------. root root system_u:object_r:sudo_db_t:s0   sudo
   [root@RHEL71 sven]# docker run -v /var/db:/data1 -i -t fedora bash
   bash-4.3# echo "test" > /data1/test.txt
   bash: /data1/test.txt: Permission denied
   bash-4.3# exit
   
This happens because the container has no knowledge of the host's SELinux
policies. 

Considering the previous example above, you can give the container write access in
any one of the following ways:

- Disable SELinux for the enture host: `setenforce 0`
- Set the directory SELinux policy to allow any container access:
  `chcon -Rt svirt_sandbox_file_t /var/db`
- Make the container `--privileged`. This disables not only SELinux constraints but
  also the default `cgroups` restrictions:
  `docker run --privileged -v /var/db:/data1 -i -t fedora`
- Disable SELinux policy constraints for this container only:
  `docker run --security-opt label:disable -v /var/db:/data1 -i -t fedora`
- Run the container processes as SELinux process type `unconfined_t`:
  `docker run --security-opt label:type:unconfined_t -v /var/db:/data1 -i -t fedora`

Any of these methods allow the container to write to the `/data1` volume and
you'll see the changes reflected on the host's `/var/db` directory.